:orphan:

.. _Chisholm_SingleCell:

Single Cell Genomes
*******************


.. |cruise| image:: /_static/catalog_thumbnails/sailboat.png
   :scale: 10%
   :align: middle


.. |globe| image:: /_static/catalog_thumbnails/globe.png
  :scale: 10%
  :align: middle


+-------------------------------+----------+----------+-------------+------------------------+----------------------+--------------+--------------+
| Dataset Name                  | Coverage | Sensor   |  Make       |     Spatial Resolution | Temporal Resolution  |  Start Date  |  End Date    |
+===============================+==========+==========+=============+========================+======================+==============+==============+
| | :ref:`Chisholm_SingleCell`  | |globe|  ||cruise|  | Observation |     Irregular          |        Irregular     |  2009-07-14  | 2016-04-27   |
+-------------------------------+----------+----------+-------------+------------------------+----------------------+--------------+--------------+

Table of Variables
******************


.. raw:: html

    <iframe src="../../_static/var_tables/tblSingleCellGenomes_Chisholm/tblSingleCellGenomes_Chisholm.html"  frameborder = 0 height = '250px' width="100%">></iframe>

|

Dataset Description
*******************

This dataset contains single cell genomes of Prochlorococcus, Synechococcus, extracellular virus particles, and additional microorganisms from a diverse range of bacterial, archaeal, and viral groups, coming from a diverse set of marine environments: Isolation information and where to access 16S rRNA, genome sequencing, and genome annotation files



Single cells were isolated from CMORE, HOT, BATS, GEOTRACES, and SCOPE cruises.



If the same "sample" was assayed multiple times (ie. technical replicates), then each replicate gets a separate row with a unique Assay Name, and the same Sample Name.



16S rRNA and 16S-23S Intergenic Transcribed Spacer (ITS) sequences: 16S ITS sequences were prepared using polymerase chain reaction (PCR) amplification of target ribosomal RNA genomic regions using group specific primers (bacterial 16S or cyanobacterial ITS) followed by sequencing using the Sanger method.



Genome sequencing and genome annotation: Libraries were prepared using Illumina Nextera XT followed by size selection (500 +/- 50 bp). Sequencing was paired-end 150+150 nt on Illumina NextSeq 500. Whole genome assembly was done using SPAdes 3.0.0. Genomes were annotated using IMG Annotation Pipeline version 4.








Data Source
***********

Submitted by the Chisholm Lab at MIT: https://chisholmlab.mit.edu/.



How to Acknowledge
******************

These single cell genomes were published (and additional information can be found) in Berube, P. M., S. J. Biller, T. Hackl, S. L. Hogle, B. M. Satinsky, J. W. Becker, R. Braakman, S. B. Collins, L. Kelly, J. Berta-Thompson, A. Coe, K. Bergauer, H. A. Bouman, T. J. Browning, D. De Corte, C. Hassler, Y. Hulata, J. E. Jacquot, E. W. Maas, T. Reinthaler, E. Sintes, T. Yokokawa, D. Lindell, R. Stepanauskas, and S. W. Chisholm. 2018. Single cell genomes of Prochlorococcus, Synechococcus, and sympatric microbes from diverse marine environments. Scientific Data 5:180154.

Version History
***************
