

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>ESV - Exact Sequence Variants &mdash; CMAP</title>
  

  
  
    <link rel="shortcut icon" href="../../_static/CMAP_logo.png"/>
  
  
  

  
  <script type="text/javascript" src="../../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../../_static/jquery.js"></script>
        <script type="text/javascript" src="../../_static/underscore.js"></script>
        <script type="text/javascript" src="../../_static/doctools.js"></script>
        <script type="text/javascript" src="../../_static/language_data.js"></script>
        <script crossorigin="anonymous" integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA=" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
        <script async="async" type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-AMS-MML_HTMLorMML"></script>
        <script type="text/x-mathjax-config">MathJax.Hub.Config({"tex2jax": {"inlineMath": [["$", "$"], ["\\(", "\\)"]], "processEscapes": true, "ignoreClass": "document", "processClass": "math|output_area"}})</script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/style.css" type="text/css" />
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home"> CMAP
          

          
            
            <img src="../../_static/CMAP_logo.png" class="logo" alt="Logo"/>
          
          </a>

          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <ul>
<li class="toctree-l1"><a class="reference internal" href="../../getting_started/installation/installation.html">Installation</a></li>
</ul>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../catalog.html">Data Catalog</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../user_guide/API_ref/api_ref.html">Software Packages</a></li>
</ul>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../../faq_and_contributing/datasubmission.html">Data Submission</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../faq_and_contributing/FAQ.html">FAQ</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../faq_and_contributing/contact.html">Contact and Contribute</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">CMAP</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html">Docs</a> &raquo;</li>
        
      <li>ESV - Exact Sequence Variants</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  
<style>
/* CSS overrides for sphinx_rtd_theme */

/* 24px margin */
.nbinput.nblast.container,
.nboutput.nblast.container {
    margin-bottom: 19px;  /* padding has already 5px */
}

/* ... except between code cells! */
.nblast.container + .nbinput.container {
    margin-top: -19px;
}

.admonition > p:before {
    margin-right: 4px;  /* make room for the exclamation icon */
}

/* Fix math alignment, see https://github.com/rtfd/sphinx_rtd_theme/pull/686 */
.math {
    text-align: unset;
}
</style>
<div class="section" id="esv-exact-sequence-variants">
<span id="esv"></span><h1>ESV - Exact Sequence Variants<a class="headerlink" href="#esv-exact-sequence-variants" title="Permalink to this headline">¶</a></h1>
<div class="section" id="exact-amplicon-sequence-variant-abundances-from-the-ant28-5-latitudinal-transect-of-the-atlantic-ocean">
<h2>Exact Amplicon Sequence Variant Abundances from the ANT28-5 Latitudinal Transect of the Atlantic Ocean<a class="headerlink" href="#exact-amplicon-sequence-variant-abundances-from-the-ant28-5-latitudinal-transect-of-the-atlantic-ocean" title="Permalink to this headline">¶</a></h2>
<table class="docutils align-default">
<colgroup>
<col style="width: 22%" />
<col style="width: 7%" />
<col style="width: 9%" />
<col style="width: 17%" />
<col style="width: 14%" />
<col style="width: 15%" />
<col style="width: 15%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Dataset Name</p></th>
<th class="head"><p>Sensor</p></th>
<th class="head"><p>Make</p></th>
<th class="head"><p>Spatial Resolution</p></th>
<th class="head"><p>Temporal Resolution</p></th>
<th class="head"><p>Start Date</p></th>
<th class="head"><p>End Date</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p><a class="reference internal" href="#esv"><span class="std std-ref">ESV - Exact Sequence Variants</span></a></p></td>
<td><p><a class="reference internal" href="../../_images/sailboat.png"><img alt="cruise" class="align-middle" src="../../_images/sailboat.png" style="width: 51.2px; height: 51.2px;" /></a></p></td>
<td><p>Observation</p></td>
<td><p>Irregular</p></td>
<td><p>Irregular</p></td>
<td><p>2012-04-11</p></td>
<td><p>2012-05-11</p></td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="dataset-description">
<h1>Dataset Description<a class="headerlink" href="#dataset-description" title="Permalink to this headline">¶</a></h1>
<p>This dataset consists of relative microbial abundances (primarily Bacteria/Archaea) sampled across a latitudinal transect of the Atlantic ocean. Samples were collected between April - May 2012 on the cruise ANT28-5 (R/V Polarstern) by the research groups of Irene Wagner-Döbler (Technische Universität Braunschweig) and Meinhard Simon (Carl von Ossietzky University of Oldenburg). At each oceanographic station, multiple depths were sampled primarily in the photic zone (≤ 200 m), and each sample was sequentially filtered to obtain three size fractions:</p>
<ol class="arabic simple">
<li><p>Free-living (0.2 - 3 μm; FL)</p></li>
<li><p>Small particle associated (3 - 8 μm; SPA)</p></li>
<li><p>Large particle associated (&gt; 8 μm; LPA)</p></li>
</ol>
<p>Detailed methods are available in three previously-published papers that discuss large-scale patterns in microbial diversity, co-occurrence, and biogeography across this transect (Milici et al., 2016a, b, c).</p>
</div>
<div class="section" id="table-of-variables">
<h1>Table of Variables<a class="headerlink" href="#table-of-variables" title="Permalink to this headline">¶</a></h1>
<iframe src="../../_static/var_tables/tblESV/tblESV.html"  frameborder = 0 height = '300px' width="100%">></iframe><div class="line-block">
<div class="line"><br /></div>
</div>
<div class="section" id="dataset-reprocessing-for-cmap">
<h2>Dataset reprocessing for CMAP:<a class="headerlink" href="#dataset-reprocessing-for-cmap" title="Permalink to this headline">¶</a></h2>
<p>In order to generate exact amplicon sequence variants (eASVs), publicly accessible amplicon data from Milici et al. (2016) was reanalyzed with qiime2 (2018.8). In brief, primers were trimmed with cutadapt, amplicons were merged with VSEARCH, and subsequently quality-filtered. eASVs were generated with deblur denoise-16s with a trim length of 231 base pairs. Taxonomic classification was carried out with the “qiime feature-classifier classify-sklearn” against the SILVA 132 database (NR99) subsetted to the amplicon region (807F = GGATTAGATACCCBRGTAGTC ; 1064R = AGYTGDCGACRRCCRTGCA). eASVs were also clustered into operational taxonomic units (OTUs) with VSEARCH at different percent sequence similarity thresholds approximately corresponding to existing taxonomic ranks (99, 98, 97, 96, 92, 89). Scripts and ancillary data are available at : <a class="reference external" href="https://osf.io/g7xsj/">https://osf.io/g7xsj/</a>.</p>
<p>Data were then transformed into relative abundances and collated with metadata provided in Milici et al (2016a, b, c) prior to ingestion into CMAP (as long as depths recorded for metadata and sequence data differed by ≤ 10 m; <a class="reference external" href="https://github.com/jcmcnch/OTUandMetadata2CMAP">https://github.com/jcmcnch/OTUandMetadata2CMAP</a>). The resulting dataset allows the user to retrieve relative abundances and environmental covariates for both individual eASVs and OTU clusters. As with all other data in CMAP, each relative abundance measurement is associated with unique time/space coordinates. Whether a given abundance is from an eASV or OTU is clearly denoted in the columns “ESV_ID_or_Cluster_Centroid”, “Clustering_Level” and “Cluster_Type”. Raw sequences of eASVs and tab-separated files are also available as part of the ancillary data allow the user to manually connect individual eASVs with OTU clusters.</p>
</div>
<div class="section" id="notes-on-data">
<h2>Notes on data:<a class="headerlink" href="#notes-on-data" title="Permalink to this headline">¶</a></h2>
<p>The primers used by Milici et al. (2016) are prokaryotic-specific, and do have mismatches to some important marine taxa (e.g. SAR11 group I). Despite these potential biases, SAR11 remain a high proportion of the total reads, and thus this bias is likely to have been small in practice. As noted in the original paper, these primers also do amplify chloroplast sequences, which are particularly abundant in the SPA and LPA fractions. In some of the particle-associated samples, there were some putatively eukaryotic sequences amplified, but they were excluded from this dataset by the positive filter of deblur.</p>
</div>
</div>
<div class="section" id="data-source">
<h1>Data Source<a class="headerlink" href="#data-source" title="Permalink to this headline">¶</a></h1>
<p>Data provided by: Jesse McNichol (<a class="reference external" href="mailto:mcnichol&#37;&#52;&#48;alum&#46;mit&#46;edu">mcnichol<span>&#64;</span>alum<span>&#46;</span>mit<span>&#46;</span>edu</a>) &amp; Jed Fuhrman lab, USC</p>
</div>
<div class="section" id="how-to-acknowledge">
<h1>How to Acknowledge<a class="headerlink" href="#how-to-acknowledge" title="Permalink to this headline">¶</a></h1>
<p>Milici, M., Tomasch, J., Wos-Oxley, M.L., Wang, H., Jáuregui, R., Camarinha-Silva, A., Deng, Z.-L., Plumeier, I., Giebel, H.-A., Wurst, M., Pieper, D.H., Simon, M., Wagner-Döbler, I., 2016a. Low diversity of planktonic bacteria in the tropical ocean. Scientific Reports 6, 19054. <a class="reference external" href="https://doi.org/10.1038/srep19054">https://doi.org/10.1038/srep19054</a></p>
<p>Milici, M., Deng, Z.-L., Tomasch, J., Decelle, J., Wos-Oxley, M.L., Wang, H., Jáuregui, R., Plumeier, I., Giebel, H.-A., Badewien, T.H., Wurst, M., Pieper, D.H., Simon, M., Wagner-Döbler, I., 2016b. Co-occurrence Analysis of Microbial Taxa in the Atlantic Ocean Reveals High Connectivity in the Free-Living Bacterioplankton. Front. Microbiol. 7. <a class="reference external" href="https://doi.org/10.3389/fmicb.2016.00649">https://doi.org/10.3389/fmicb.2016.00649</a></p>
<p>Milici, M., Tomasch, J., Wos-Oxley, M.L., Decelle, J., Jáuregui, R., Wang, H., Deng, Z.-L., Plumeier, I., Giebel, H.-A., Badewien, T.H., Wurst, M., Pieper, D.H., Simon, M., Wagner-Döbler, I., 2016c. Bacterioplankton Biogeography of the Atlantic Ocean: A Case Study of the Distance-Decay Relationship. Front. Microbiol. 7. <a class="reference external" href="https://doi.org/10.3389/fmicb.2016.00590">https://doi.org/10.3389/fmicb.2016.00590</a></p>
</div>
<div class="section" id="version-history">
<h1>Version History<a class="headerlink" href="#version-history" title="Permalink to this headline">¶</a></h1>
</div>


           </div>
           
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2019, Mohammad Dehghani Ashkezari &amp; Norland Raphael Hagen
      <span class="lastupdated">
        Last updated on Jan 13, 2020.
      </span>

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>